guides/hosting/infrastructure/elasticsearch/elasticsearch-setup.md

```diff
@@ -164,3 +164,24 @@ If you want more than that, a tool like [RabbitMq](/docs/guides/hosting/infrastr
 
 Keep in mind that the search configuration of Shopware has no effect when using Elasticsearch.
 To configure which fields and elements are searchable when using Elasticsearch, you must install the extension [Advanced Search](https://docs.shopware.com/en/shopware-6-en/enterprise-extensions/enterprise-search).
+
+## Elasticsearch for Admin
+
+In Shopware 6.4.19.0, the "AND/OR Search" feature was released to be used in Administration.
+
+Add the below config variables to set up Elasticsearch for Administration:
+
+```bash
+ADMIN_OPENSEARCH_URL=YOUR OPEN SEARCH URL 
+SHOPWARE_ADMIN_ES_ENABLED=1
+SHOPWARE_ADMIN_ES_REFRESH_INDICES=1
+SHOPWARE_ADMIN_ES_INDEX_PREFIX=sw-admin
+```
+
+Also the CLI Commands used are as below:
+
+```bash
+bin/console es:admin:index                                              
+bin/console es:admin:reset                                              
+bin/console es:admin:test
+```
```

Comment:
Maybe we can add a hint for the advanced admin users, under the hood we are using https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-simple-query-string-query.html to search so the advanced user can utilize it in their queries